sessionctx/variable/session.go

```diff
@@ -828,6 +830,14 @@ type SessionVars struct {
 	// CTEMaxRecursionDepth indicates The common table expression (CTE) maximum recursion depth.
 	// see https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_cte_max_recursion_depth
 	CTEMaxRecursionDepth int
+
+	TemporaryTable
+}
+
+// TemporaryTable is defined for the temporary table feature.
+type TemporaryTable struct {
+	// Global records the global temporary table used by the transaction.
+	Global map[int64]struct{}
```

Comment:
Now that global temporary tables are recorded in `TransactionContext', why do we need this here?